From 5f35d0b2975cb331858716f86cffa817f3712957 Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@st.com>
Date: Mon, 20 Feb 2023 10:08:03 +0100
Subject: [PATCH 0267/1141] arm64: defconfig: add CAN configs in addons
 fragment

Add CAN1/CAN3 peripherals config flags in addons fragment.

Signed-off-by: Christophe Roullier <christophe.roullier@st.com>
Change-Id: Ic89ac8e6aa1b72348de96b35d753cc47197a78d4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/291671
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/310721
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
Domain-Review: Alexandre TORGUE <alexandre.torgue@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 1a6bc3f655a5..bc661d192e90 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -2,6 +2,8 @@
 CONFIG_ARCH_STM32=y
 CONFIG_ARM_SCMI_CPUFREQ=y
 CONFIG_ARM_SMC_WATCHDOG=y
+CONFIG_CAN_M_CAN=m
+CONFIG_CAN_M_CAN_PLATFORM=m
 CONFIG_COUNTER=m
 CONFIG_DWMAC_GENERIC=y
 CONFIG_DWMAC_STM32=y
-- 
2.39.2

